db/seeds.rb

```diff
@@ -1,7 +1 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+load(Rails.root.join('db', 'seeds', "#{Rails.env.downcase}.rb"))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.